File: 비주얼 베이직/비베 기말/2019--1-11주차-자작함수-김유빈/2019--1-11주차-자작함수-김유빈/2019--1-11주차-자작함수-김유빈/Form2.vb
Public Class Form2
    Private Sub Form2_Load(sender As Object, e As EventArgs) Handles MyBase.Load

        PictureBox1.Image = Image.FromFile(CurDir() & "\blue.ico")

    End Sub

    Private Sub ChangeSignal(ByVal Color As Byte)
        Select Case Color
            Case 1
                PictureBox1.Image = Image.FromFile(CurDir() & "\red.ico")
            Case 2
                PictureBox1.Image = Image.FromFile(CurDir() & "\yellow.ico")
            Case 3
                PictureBox1.Image = Image.FromFile(CurDir() & "\blue.ico")
        End Select
    End Sub
    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click

        Call ChangeSignal(1) '빨강
    End Sub

    Private Sub Button2_Click(sender As Object, e As EventArgs) Handles Button2.Click
        Call ChangeSignal(2) '노랑
    End Sub

    Private Sub Button3_Click(sender As Object, e As EventArgs) Handles Button3.Click
        Call ChangeSignal(3) '녹색
    End Sub
End Class

File: C#/2019-2-C샵-7주차-자작함수와신호등만들기-김유빈/2019-2-C샵-7주차-자작함수와신호등만들기-김유빈/Form2.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace _2019_2_C샵_7주차_자작함수와신호등만들기_김유빈
{
    public partial class Form2 : Form
    {
        public Form2()
        {
            InitializeComponent();
        }
        public void DisplaySinhodoong(int a)
        {
            switch (a)
            {
                case 1:
                    pictureBox1.Image = Image.FromFile
               (System.Environment.CurrentDirectory + "/red.ico");
                    break;
                case 2:
                    pictureBox1.Image = Image.FromFile
               (System.Environment.CurrentDirectory + "/yellow.ico");
                    break;
                case 3:
                    pictureBox1.Image = Image.FromFile
               (System.Environment.CurrentDirectory + "/blue.ico");
                    break;
            }
        }

        private void Form2_Load(object sender, EventArgs e)
        {
            DisplaySinhodoong(3);
        }

        private void button1_Click(object sender, EventArgs e)
        {
            DisplaySinhodoong(1);
        }

        private void button2_Click(object sender, EventArgs e)
        {
            DisplaySinhodoong(2);
        }

        private void button3_Click(object sender, EventArgs e)
        {
            DisplaySinhodoong(3);
        }
    }
}
